tests/numerics/ad/test_operators.py

```diff
@@ -651,6 +629,170 @@ def _compare_ad_objects(a, b):
     assert np.allclose(true_state[ind1], v1_prev.value(eq_system, true_iterate))
 
 
+def test_ad_variable_prev_iter():
+    # Test only 1 variable, the rest should be covered by other tests
+    mdg, _ = pp.mdg_library.square_with_orthogonal_fractures(
+        "cartesian",
+        {"cell_size": 0.2},
+        fracture_indices=[1],
+    )
+    eqs = pp.ad.EquationSystem(mdg)
+
+    # Integer to test the depth of prev _*, could be a test parameter, but no need
+    depth = 5
```

Comment:
General comment: Do we gain anything from having depth = 5 and not, say, 2 or 3? If not, keeping it lower will somewhat limit the cost of the test. In the same spirit, can we set cell_size to 0.5 and have (on my count) 6 dofs in total instead of 30?

Reply:
You are right, changed it.